sklearn/utils/estimator_checks.py

```diff
@@ -335,10 +337,24 @@ def _construct_instance(Estimator):
                 estimator = Estimator(Ridge())
             else:
                 estimator = Estimator(LinearDiscriminantAnalysis())
+        elif required_parameters in (['estimators'],):
+            # Heterogeneous ensemble classes (i.e. stacking, voting)
+            if issubclass(Estimator, RegressorMixin):
```

Comment:
except that just because the meta is a regressor, it doesn't mean it consists of  regressors...

Reply:
I would say that it could be a reasonably default for the moment (we can add a meaningful comment).
Right now we are not testing every combination in common tests even for other estimators so it might be a start?